packages/lib-utils/src/k8s/k8s-utils.ts

```diff
@@ -207,18 +207,17 @@ export const getGroupVersionKindForModel: GetGroupVersionKindForModel = ({
  * @deprecated - This will become obsolete when we move away from K8sResourceKindReference to K8sGroupVersionKind
  * Provides a reference string that uniquely identifies the group, version, and kind of a k8s resource.
  * @param K8sGroupVersionKind Pass K8sGroupVersionKind which will have group, version, and kind of a k8s resource.
- * @param K8sGroupVersionKind.apiGroup Pass group of k8s resource or model.
- * @param K8sGroupVersionKind.apiVersion Pass version of k8s resource or model.
+ * @param K8sGroupVersionKind.group Pass group of k8s resource or model.
+ * @param K8sGroupVersionKind.version Pass version of k8s resource or model.
  * @param K8sGroupVersionKind.kind Pass kind of k8s resource or model.
  * @return The reference for any k8s resource i.e `group~version~kind`.
  * If the group will not be present then "core" will be returned as part of the group in reference.
  * * */
 export const getReference = ({
-  apiGroup,
-  apiVersion,
+  group,
+  version,
   kind,
-}: K8sResourceIdentifier): K8sResourceKindReference =>
-  [apiGroup || 'core', apiVersion, kind].join('~');
+}: K8sGroupVersionKind): K8sResourceKindReference => [group || 'core', version, kind].join('~');
```

Comment:
@florkbr Not sure what you were up to with this change... but I have reverted it. `K8sResourceIdentifier` and `K8sGroupVersionKind` are not the same type.

Reply:
Yep. Thanks! Not enough coffee ☕